packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -710,7 +748,7 @@ export const Table = <
                         key={columnProps.key}
                         title={undefined}
                         ref={(el) => {
-                          if (el && isResizable) {
+                          if (el) {
```

Comment:
why removed?

Reply:
I need to get actual column widths to use them for `left` and `right` css

Reply:
you mean, you need columns ref only

Reply:
Using it here:
![image](https://user-images.githubusercontent.com/36186912/171824463-66050755-ba82-4113-933c-a8014bb8ecfa.png)